rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/NodeSourceWindow.java

```diff
@@ -650,12 +650,14 @@ private void populateHiddenItemsIfNecessary(PluginDescriptor pluginDescriptor) {
                     }
                 });
             } else if (pluginField.isCheckbox()) {
-                CheckboxItem checkboxItem = new CheckboxItem(plugin.getPluginName() + pluginField.getName(),
-                                                             pluginField.getName());
-                checkboxItem.setShowTitle(true);
-                checkboxItem.setShowLabel(false);
-                checkboxItem.setLabelAsTitle(true);
-                formItemsForField.add(checkboxItem);
+                LinkedHashMap<String, String> trueFalseKeyValue = new LinkedHashMap<>();
```

Comment:
We could add a comment to explain why using radio group for checkBox fields (It will be usable for the future developers).

Reply:
you are right